vm-setup/roles/v1aX_integration_test/tasks/download_image.yml

```diff
@@ -32,22 +39,24 @@
           content: |
             {{ centos_md5.stat.checksum }}
 
-          dest: "{{ IRONIC_IMAGE_DIR }}/{{ IMAGE_NAME_CENTOS }}.md5sum"
+          dest: "{{ IRONIC_IMAGE_DIR }}/{{ RAW_IMAGE_NAME_CENTOS }}.md5sum"
           mode: 0664
     when:
       - image_path_centos.stat.exists == False
-      - IMAGE_OS == "Centos"
```

Comment:
this would be problematic, since image_path_centos would not exist if IMAGE_OS is not Centos. this whole block would need to be nested under the previous block checking whether IMAGE_OS is Centos